workflows/alevin-quant/run-alevin-fry.nf

```diff
@@ -88,7 +96,7 @@ process generate_permit{
       -i ${run_dir} \
       --expected-ori fw \
       -o ${run_dir} \
-      ${params.unfiltered ? "-u ${barcode_file}" : '--knee-distance'}
+      ${params.filter == 'knee' ? '--knee-distance' :  "-u ${barcode_file}"}
```

Comment:
I think we might want to move the filtering parameter options to the top of the script as a mapping/dictionary, as we have it in the `alevin` process. The reason is that we might want more flexibility if more filtering options come later. I am also not a huge fan of ternary operators in the case where somebody might input an unexpected value.